controllers/components/kserve/kserve_controller.go

```diff
@@ -150,7 +149,6 @@ func (s *componentHandler) NewComponentReconciler(ctx context.Context, mgr ctrl.
 		WithAction(deploy.NewAction(
 			deploy.WithCache(),
 		)).
-		WithAction(security.NewUpdatePodSecurityRoleBindingAction(serviceAccounts)).
```

Comment:
This is not needed any longer?

Reply:
According to the code, the `serviceAccounts` list was only including model controller related info, so since there is a dedicated component, it should not be needed here.

Reply:
But this is not part of manifest refactoring, isn't it?

Reply:
not strictly, but since some constants such as the model controller component name constant were moved, it resulted in an error so removed it